api_spec.yaml

```diff
@@ -1743,6 +1747,8 @@ paths:
                   $ref: '#/components/schemas/token'
                 userDataInJWT:
                   $ref: '#/components/schemas/userDataInJWT'
+                grants:
+                  $ref: '#/components/schemas/grantPayload'
```

Comment:
The contents of the grant will completely override existing content in the db for this session - correct?

Reply:
Yeah, we could do something like the shallow merging I proposed for the user metadata API, but since they already have access to the token (and the grants), we can do the merging on the backend side. This will override stuff was added using the session handle though.

Reply:
Yea. So we can maintain the same semantics here as `userDataInJWT`. That is, fully override. One point to note is that `grants` and `userDataInJWT` should be optional in this API. If they are not present, they are not updated. As opposed to them being present, but being empty objects. In which case, they will be set to empty objects